.github/workflows/ci.yml

```diff
@@ -0,0 +1,106 @@
+name: CakePHP CI
```

Comment:
Please keep this as `CI` for consistency across all repos. The workflow name is used to URL to filter the results on Actions page, which are needed for "build status" badges.

Reply:
I just copied the .github folder from another repo. Someone will have to follow up and check them all I think.

Reply:
I'll do that whenever I get some time.